packages/react-scripts/scripts/init.js

```diff
@@ -130,6 +130,9 @@ module.exports = function(
       build: 'sg1-module-scripts build',
       test: 'sg1-module-scripts test',
       'test:cov': 'npm run test -- --coverage --watchAll=false',
+      'test:e2e': 'cypress open',
+      graphql: 'graphql-codegen --config codegen.yml',
```

Comment:
🔥 yes to happy to add both of these

Reply:
should we add a default codegen.yml?

Reply:
Yes, I am adding that as well as the drone file.